packager/util/config.go

```diff
@@ -200,7 +202,15 @@ func ReadConfig(configBytes []byte) (*Config, error) {
 	// TODO(twifkak): Return an error if the TOML includes any fields that aren't part of the Config struct.
 
 	if config.Port == 0 {
-		config.Port = 8080
+		if os.Getenv("PORT") != "" {
```

Comment:
(nit) Can you make a local var for `os.Getenv("PORT")` so you don't have to call it again on lines 206 and 209? Not an actual perf or correctness thing, just an aesthetic thing.